app/services/recommendation_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
```

Comment:
Should this be inside `models/`?
Not sure it's a service as such

Reply:
yes, I think you're right!